### ISSUE TITLE
The interface to warnnier90 is incorrect.

### ISSUE THREAD

**Issue by pxlxingliang** [deepmodeling/abacus-develop#3418]
### Describe the bug

I try to run the example in examples/interface_wannier90 in [code](https://github.com/deepmodeling/abacus-develop/tree/develop/examples/interface_wannier90), and when I execute the last step: `wannier90.x diamond.win`, the wannier90 program will throw the error message:
```
 plot_wannier: file UNK00001.1  not found
Error: examine the output/error file for details
```
And files UNK00001.* are not generated in OUT.ABACUS. 

Should add some extra parameter to generate these files? 

### Task list for Issue attackers (only for developers)

- [ ] Verify the issue is not a duplicate.
- [ ] Describe the bug.
- [ ] Steps to reproduce.
- [ ] Expected behavior.
- [ ] Error message.
- [ ] Environment details.
- [ ] Additional context.
- [ ] Assign a priority level (low, medium, high, urgent).
- [ ] Assign the issue to a team member.
- [ ] Label the issue with relevant tags.
- [ ] Identify possible related issues.
- [ ] Create a unit test or automated test to reproduce the bug (if applicable).
- [ ] Fix the bug.
- [ ] Test the fix.
- [ ] Update documentation (if necessary).
- [ ] Close the issue and inform the reporter (if applicable).

**Comment by jingan-181** [comment]
1. In ABACUS 3.5.0, since the default parameter of `out_wannier_unk` is changed to 0, "UNK.*" files related to wave functions will not be output. `out_wannier_unk 1` needs to be added manually in ` examples/interface_wannier90`.
2. In ABACUS 3.5.0, when the basis set used is numerical atomic orbitals and `wannier_method  2`, the `out_wannier_unk` parameter will be invalid. This will be added in future version.

**Comment by Liu-RX** [comment]
The UNK files are only necessary if you want to plot the Wannier function isosurface. Only the mmn file is indispensable in case of maximally localization of Wannier.

**Comment by pxlxingliang** [comment]
Hi @jingan-181, I use the latest codes to run example `ABACUS_towannier90_lcao`, the final WF centre and spread are much different with that by v3.4.2. Is it reasonable?

by v3.4.2
![image](https://github.com/deepmodeling/abacus-develop/assets/91927439/9e2059dc-96da-49a8-b99e-0867850f452b)

by latest codes:
![image](https://github.com/deepmodeling/abacus-develop/assets/91927439/d096e308-7351-4401-b490-d6c4d184678b)

**Comment by jingan-181** [comment]
It is incorrect for spread to be a negative number. I will check the interface for bugs as soon as possible.

**Comment by dyzheng** [comment]
The latest report is band structure calculated by ABACUS-wannier90-interface differ from which calculated by ABACUS. It is a severe bug.

**Comment by Liu-RX** [comment]
OK. I will first try to solve this.

---- Replied Message ----
From dyzheng ***@***.***> Date 01/31/2024 11:11 To deepmodeling/abacus-develop ***@***.***> Cc Liu Renxi ***@***.***>,
Comment ***@***.***> Subject Re: [deepmodeling/abacus-develop] The interface to warnnier90 is incorrect. (Issue #3418)  

The latest report is band structure calculated by ABACUS-wannier90-interface differ from which calculated by ABACUS. It is a severe bug.

**Comment by Liu-RX** [comment]
Is it due to the change of the LCAO mmn calculation method? The implementation of LCAO basis mmn in 3.4.2 is still the old method, while 3.5.0 implements the new method. @jingan-181

**Comment by shiruosong** [comment]
Has the problem been solved?

When I used version 3.5.3 to calculate wannier center based on lcao, the spread calculated by many structures would be too large, even tens. Therefore, bands calculated by wannier are also different from those calculated directly by ABACUS.  There is no problem with vasp or qe for the same structure.

I reproduced the problem in this issue when testing examples, so it feels like it may be the same problem I encountered.
